dev-tools/notice/rules.json

```diff
@@ -9,7 +9,8 @@
     "ISC",
     "MIT",
     "MPL-2.0",
-    "Public Domain"
+    "Public Domain",
+    "Zlib"
```

Comment:
Yes its greenlight. For other reviewers going through this.

Reply:
I wonder if we should add this in a separate PR to have it "documented" in case someone looks for the diff so it does not look like "accidentally" added.

Reply:
I can move it to a single PR just for this one thing. Might be weird to add in an individual PR, think it makes better sense in context.